src/ychaos/agents/system/diskFill.py

```diff
@@ -0,0 +1,104 @@
+#  Copyright 2021, Yahoo
+#  Licensed under the terms of the Apache 2.0 license. See the LICENSE file in the project root for terms
+import math
+import shutil
+from pathlib import Path
+from queue import LifoQueue
+
+from pydantic import Field
+
+from ..agent import (
+    Agent,
+    AgentMonitoringDataPoint,
+    AgentPriority,
+    TimedAgentConfig,
+)
+from ..utils.annotations import log_agent_lifecycle
+
+
+class DiskFillConfig(TimedAgentConfig):
+    """
+    Defines the Disk Fill configuration to consume the disk space. The framework will
+    fill the disk space of a directory defined in `partition` and fill only a percentage of available
+    space, as defined in the `partition_pct` attribute. By default,
+    `partition_pct` is set to 100, implying the complete disk space of that partition will be filled.
+    """
+
+    name = "disk_fill"
+    description = "This agent fills up disk space."
+
+    priority = AgentPriority.MODERATE_PRIORITY
+
+    partition: Path = Field(
+        description="The Filepath of directory or partition to fill",
+        default=Path("/etc/"),
```

Comment:
Let us add some examples here
```suggestion
        default=Path("/etc/"),
        examples=["/tmp", "/home/tmpuser"]
```